packages/backend/src/models/message.ts

```diff
@@ -3,9 +3,10 @@ import { MODEL_MESSAGE } from '@constant/general'
 import { MessageSchema } from '@model/schemas/message'
 
 export interface MessageInterface {
-  author: string
-  avatarUrl: string
+  creator: string
+  avatarUrl: string //TODO: utilize this if needed, if not just remove it from model
```

Comment:
Yeah i had problems integrating this with the designs... I had considered instead giving people a few choices of decoration elements for the top right of the card. (This is nothing to change, I just put this here for future reference)

Reply:
So is it better to remove the `avatarUrl` or just keep it for now?

Reply:
Hmm i'd like to hear the opinions of the others on this

Reply:
So we decided that keeping this doesn't make sense right now.